package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@comparaonline/event-streamer",
-  "version": "4.1.0",
```

Comment:
I think it's better to increment the minor (4.2.0) because you added a new feature maintaining compatibility with previous versions.